packages/next/src/server/use-cache/use-cache-wrapper.ts

```diff
@@ -697,10 +697,16 @@ export function cache(
           workUnitStore === undefined || workUnitStore.type === 'unstable-cache'
             ? []
             : workUnitStore.implicitTags
-        const entry: undefined | CacheEntry = await cacheHandler.get(
-          serializedCacheKey,
-          implicitTags
-        )
+
+        const forceRevalidate =
+          workStore.dev &&
+          workUnitStore?.type === 'request' &&
```

Comment:
I think this means that when we generate a fresh outer cache entry it is still possible for inner caches to be used. I think that's not quite equivalent to what we had before with unstable_cache and fetch cache (where there are no inner caches)

Probably we need to apply this forceRevalidate regardless of workUnitStore type

Reply:
Fixed in [1e02431](https://github.com/vercel/next.js/pull/75754/commits/1e02431ac40037c2056a0e84946908122e89bb7b).